### PR TITLE
Translate concepts/overview/kubernetes-api in Korean

### LINKED ISSUE #10718: Translate concepts/overview/kubernetes-api in Korean
**This is a...** 

- [x] Feature Request
- [ ] Bug Report

**Problem:**
No concepts/overview/kubernetes-api in Korean

**Proposed Solution:**
Translate concepts/overview/kubernetes-api in Korean

**Page to Update:**
https://kubernetes.io/ko/docs/concepts/overview/kubernetes-api/

### PR DESCRIPTION
**Pull request by gochist**
fixes #10718

### CODE REVIEW

File: content/ko/docs/concepts/overview/kubernetes-api.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+API 버전 규칙과 소프트웨어 버전 규칙은 간접적으로 연관되어 있음을 알아두자. [API and release versioning proposal](https://git.k8s.io/community/contributors/design-proposals/release/versioning.md)에는 API 버전 규칙과 소프트웨어 버전 규칙 간의 관계가 기술되어 있다.
+
+
+API 버전이 다른 경우는 안정성이나 기술 지원의 수준이 다르다는 것을 암시한다. 각각의 수준에 대한 조건은 [API Changes documentation](https://git.k8s.io/community/contributors/devel/api_changes.md#alpha-beta-and-stable-versions)에서 상세히 다룬다. 요약하자면 다음과 같다.
+
+- 알파(Alpha) 수준:
+  - 버전 이름에 `alpha`가 포함된다. (예: `v1alpha1`)
+  - 버그가 있을 수도 있다. 이 기능을 활성화하면 버그가 노출될 수 있다. 기본적으로 비활성화되어 있다.
+  - 기능에 대한 기술 지원이 언제든 공지 없이 중단될 수 있다.
```
Comment (ianychoi) [inline content/ko/docs/concepts/overview/kubernetes-api.md]: "공지 없이" -> "공지없이"

Comment (gochist) [reply content/ko/docs/concepts/overview/kubernetes-api.md]: "공지 없이"가 맞춤법에 맞는 표현 아닌가요? https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=60&qna_seq=128837&pageIndex=1

Comment (ianychoi) [reply content/ko/docs/concepts/overview/kubernetes-api.md]: @gochist "공지없이" on the next line is different. Please make them consistent.

Comment (gochist) [reply content/ko/docs/concepts/overview/kubernetes-api.md]: @ianychoi Thank you! I addressed your review.

**Pull request merged**